asdf/yamlutil.py

```diff
@@ -310,9 +309,6 @@ def _walker(node):
         tag_type = ctx.type_index.from_yaml_tag(ctx, tag, _serialization_context=_serialization_context)
         # This means the tag did not correspond to any type in our type index.
         if tag_type is None:
-            if not ctx._ignore_unrecognized_tag:
-                warnings.warn("{} is not recognized, converting to raw Python "
```

Comment:
I think we ought to keep this warning, just remove the flag that disables it.  Users can hide the warning with the built-in warnings module.

Reply:
See 1bf58ce.